EasyPost/Models/API/Claim.cs

```diff
@@ -0,0 +1,182 @@
+using System.Collections.Generic;
+using System.Linq;
+using EasyPost._base;
+using EasyPost.Models.Shared;
+using EasyPost.Utilities.Internal;
+using Newtonsoft.Json;
+
+namespace EasyPost.Models.API
+{
+#pragma warning disable CA1724 // Naming conflicts with Parameters.Claim
+    /// <summary>
+    ///     Class representing a <a href="https://www.easypost.com/docs/api#claim-object">EasyPost claim object</a>.
+    /// </summary>
+    public class Claim : EasyPostObject, Parameters.IClaimParameter
+    {
+        #region JSON Properties
+
+        /// <summary>
+        ///     The amount that has been approved for reimbursement.
+        /// </summary>
+        [JsonProperty("approved_amount")]
+        public string? ApprovedAmount { get; set; }
+
+        /// <summary>
+        ///    The attachments associated with the claim.
+        /// </summary>
+        [JsonProperty("attachments")]
+        public List<string>? Attachments { get; set; }
+
+        /// <summary>
+        ///     The address to which the reimbursement check should be sent.
+        /// </summary>
+        [JsonProperty("check_delivery_address")]
+        public string? CheckDeliveryAddress { get; set; }
+
+        /// <summary>
+        ///     The email address of the contact person for the claim.
+        /// </summary>
+        [JsonProperty("contact_email")]
+        public string? ContactEmail { get; set; }
+
+        /// <summary>
+        ///     A detailed description of the claim.
+        /// </summary>
+        [JsonProperty("description")]
+        public string? Description { get; set; }
+
+        /// <summary>
+        ///     A list of <see cref="ClaimHistoryEntry"/>s representing the history of the claim.
+        /// </summary>
+        [JsonProperty("history")]
+        public List<ClaimHistoryEntry>? History { get; set; }
+
+        /// <summary>
+        ///     The original amount of the insurance on the associated <see cref="Models.API.Shipment"/>.
+        /// </summary>
+        [JsonProperty("insurance_amount")]
+        public double? InsuranceAmount { get; set; }
+
+        /// <summary>
+        ///     The ID of the <see cref="Models.API.Insurance"/> object associated with this claim.
+        /// </summary>
+        [JsonProperty("insurance_id")]
+        public string? InsuranceId { get; set; }
+
+        // ReSharper disable once InconsistentNaming
+#pragma warning disable SA1300
+        [JsonProperty("payment_method")]
+        private string? _paymentMethod { get; set; }
+#pragma warning restore SA1300
+
+        /// <summary>
+        ///     The <see cref="ClaimPaymentMethod"/> of the claim.
+        /// </summary>
+#pragma warning disable SA1101
+        [JsonIgnore]
+        public ClaimPaymentMethod? PaymentMethod => _paymentMethod == null ? null : ValueEnum.FromValue<ClaimPaymentMethod>(_paymentMethod);
+#pragma warning disable SA1101
```

Comment:
I don't think we need to have a model class for this, we can just define it as a string.